### PR TITLE
EZEE-2062: Unable to embed second item in Richtext field

### PR DESCRIPTION
**Pull request by tischsoic**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZEE-2062 <br /> https://jira.ez.no/browse/EZP-28994 <br /> https://jira.ez.no/browse/EZP-28835
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes/no
| Doc needed?   | yes/no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Partially fixes problem. Partially because when you have empty richtext and then:
1. add new embed
2. delete paragraph above

the Add Toolbar will not be displayed. This is because there is no selection region on Ebed field. Style Toolbar is displayed because it is not positioned based on selection.region directly. 

It fixes situation when nothing is selected, but editor has focus. 

#### Checklist:
- [ ] ~Coding standards (`$ composer fix-cs`)~
- [x] Ready for Code Review

### CODE REVIEW

**Comment by lserwatka** [comment]
@sunpietro can we proceed with QA here?

File: src/bundle/Resources/public/js/alloyeditor/src/toolbars/ez-add.js [hunk 1]
```diff
@@ -7,34 +7,60 @@ export default class EzToolbarAdd extends AlloyEditor.Toolbars.add {
...
     /**
      * Lifecycle. Renders the UI of the button.
      *
      * @method render
-     * @return {Object} The content which should be rendered.
+     * @returns {Object} The content which should be rendered.
      */
     render() {
         const selectionData = this.props.selectionData;
+        const nothingSelected = this.checkIsNothingSelected();
```
Comment (sunpietro) [inline src/bundle/Resources/public/js/alloyeditor/src/toolbars/ez-add.js]: Nitpick: this variable is not required. The function itself is pretty self-explanatory, so I would avoid declaring this variable.

**Comment by lserwatka** [comment]
@tischsoic @dew326 what is the status here?

**Comment by tischsoic** [comment]
@lserwatka, I think that @dew326 is right. I think we may close this PR and try other solution in other PR.

**Comment by tischsoic** [comment]
Closing this PR, because one issue was solved in: https://github.com/ezsystems/ezplatform-admin-ui/pull/745 and for the 2nd we need different approach.

**Pull request closed without merging**